packages/components/src/Icons/bin/wrapSVGs.spec.ts

```diff
@@ -0,0 +1,43 @@
+// @ts-ignore: Redeclared module error
+const { insertSvgData, svgToComponentTitle } = require("./wrapSVGUtils")
+
+describe("svgToComponentTitle", () => {
+  it("converts kebab case to pascal case, and replaces .icon with Icon", () => {
+    const original = "academy.icon"
+    const expected = "AcademyIcon"
+    expect(svgToComponentTitle(original)).toEqual(expected)
+  })
+  it("handles multiple dashes in kebab case", () => {
+    const original = "action-off-white.icon"
+    const expected = "ActionOffWhiteIcon"
+    expect(svgToComponentTitle(original)).toEqual(expected)
+  })
+})
+
+describe("insertSvgData", () => {
+  const reactTemplate = `
+    import React from "react"
+    import { SVG, SVGProps } from "~components/Icons/subComponents/SVG"
+
+    export const COMPONENT_TITLE = (props: Omit<SVGProps, "children">): JSX.Element => (
+      <SVG {...props}>SVG_CONTENT</SVG>
+    )
+  `
+  const componentTitle = "MyIcon"
+  const svgContent = "<use></use>"
+  const result = insertSvgData(reactTemplate, componentTitle, svgContent)
+
+  it("replaces COMPONENT_TITLE with the name of the icon component", () => {
+    expect(result.includes("COMPONENT_TITLE")).toBe(false)
+    expect(result.includes(componentTitle)).toBe(true)
+  })
+
+  it("replaces SVG_CONTENT with the contents of the svg", () => {
+    expect(result.includes("SVG_CONTENT")).toBe(false)
+    expect(result.includes(svgContent)).toBe(true)
+  })
+  it("replaces SVG_CONTENT with the contents of the svg", () => {
+    expect(result.includes("SVG_CONTENT")).toBe(false)
+    expect(result.includes(svgContent)).toBe(true)
+  })
```

Comment:
I think a better test would be to have a single expect that has the whole formed component.
If possible, read from the template file as well, so we can ensure this is an accurate test.
Something like this:

```
const reactTemplate = templateStringFromFile

it("correctly generates component from template", () => {
  const result = insertSvgData(reactTemplate, "MyIcon", "<use></use>")
  const expected = `
    import React from "react"
    import { SVG, SVGProps } from "~components/Icons/subComponents/SVG"
    export const MyIcon = (props: Omit<SVGProps, "children">): JSX.Element => (
      <SVG {...props}><use></use></SVG>
    )
  `
  expect(result).toBe(expected)
})
```